content/browser/child_process_launcher_helper.cc

```diff
@@ -96,8 +97,22 @@ ChildProcessLauncherHelper::ChildProcessLauncherHelper(
   relaunch_renderer_process_monitor_timeout_.reset(new TimeoutMonitor(
       base::Bind(&ChildProcessLauncherHelper::OnCastanetsRendererTimeout,
                  base::Unretained(this))));
-  relaunch_renderer_process_monitor_timeout_->Start(
-      base::TimeDelta::FromSeconds(kTcpConnectionTimeout));
+  if (base::CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kTcpLaunchTimeout)) {
+    int tcp_launch_timeout;
```

Comment:
How about this?
int tcp_launch_timeout = kTcpLaunchTimeoutDefault;
and unify two different parameters of |timeout_->Start|.
'else' may not be necessary.

Reply:
When base::StringToInt gets empty string with only --tcp-launch-timeout switch not being with its timeout value,
then it returns false setting tcp_launch_timeout to '0'.

So in that case if we declare tcp_launch_timeout; as below,

int tcp_launch_timeout = kTcpLaunchTimeoutDefault;

then it will be overriden by 0, I'm afraid.

But if any better solution, please suggest it:)